dev-packages/application-package/src/application-package.ts

```diff
@@ -58,7 +61,7 @@ export class ApplicationPackage {
     }
 
     get target(): ApplicationProps.Target {
-        return this.props.target;
+        return this._appTarget || this.props.target;
```

Comment:
How about overriding it in `this._props` at the instantiation time?

Reply:
Yeah. I thought about that. I thought maybe it would be useful to have the untouched package.json values. But I should probably err on the side of less props though.